.editorconfig

```diff
@@ -312,8 +312,8 @@ csharp_prefer_simple_default_expression = true:suggestion
 csharp_style_deconstructed_variable_declaration = true:suggestion
 csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_prefer_local_over_anonymous_function = false:suggestion # not default, default is true, avoided because we use anonymous functions in multiple places and it does not make the code clearer
-csharp_style_prefer_index_operator = true:suggestion
-csharp_style_prefer_range_operator = true:suggestion
+csharp_style_prefer_index_operator = false:suggestion # not default, default is true, disabled because Index operator is not supported in all the TFMs we support.
```

Comment:
Do you know which tfms don't support it? It would be worth mentioning it here so we don't have to investigate the motifvation for this comment when we move on to next version of netfx.

Reply:
Added supported versions.